.github/workflows/main.yml

```
@@ -1,35 +1,102 @@
 on:
-  pull_request:
-    types: [opened, synchronize]
+  workflow_dispatch:
+    inputs:
+      dispatch_token:
+        description: 'Token that authorize the dispatch'
+        required: true
+      head_sha:
+        description: 'Head commit SHA that dispatched the workflow'
+        required: true
+      pr_author_username:
+        description: 'Pull Request author username'
+        required: true
+      pr_number:
+        description: 'Pull Request number that dispatched the workflow'
+        required: true
+
 jobs:
-  eslint:
-    runs-on: ubuntu-18.04
+  ESLint-Evaluator:
+    runs-on: self-hosted
+    name: ESLint
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1.4.4
+      - name: Fetch project repository
+        uses: actions/checkout@v2
+
+      - name: Fetch Blocked Files Checkout action
+        uses: actions/checkout@v2
+        with:
+          repository: betrybe/blocked-files-checkout-action
+          ref: v2
+          token: ${{ secrets.GIT_HUB_PAT }}
+          path: .github/actions/blocked-files-checkout
+
+      - name: Fetch ESLint evaluator
+        uses: actions/checkout@v2
+        with:
+          repository: betrybe/eslint-linter-action
+          ref: v3
+          token: ${{ secrets.GIT_HUB_PAT }}
+          path: .github/actions/eslint-evaluator
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@v1.4.4
         with:
           node-version: '14'
-      - name: ESLint
-        uses: betrybe/eslint-linter-action@v2.1
+
+      - name: Restore protected files
+        uses: ./.github/actions/blocked-files-checkout
+        with:
+          restore_branch: 'master'
```

```suggestion
          restore_branch: 'main'
```
Esse repositório de rubrica está usando `main` diferente dos outros 🤔. Acredito que precisa trocar para ele conseguir recuperar. Isso se repete no **Restore protected files** do StyleLint e do Evaluator.

---

Bem observado @SamuelSilvaMelo.
Alterei agora.